toonz/sources/toonz/mainwindow.cpp

```diff
@@ -1758,8 +1758,8 @@ void MainWindow::defineActions() {
                          "Alt+L");
   createRightClickMenuAction(MI_ToggleXSheetToolbar,
                              tr("Toggle XSheet Toolbar"), "");
-  createRightClickMenuAction(MI_ToggleXsheetCameraColumn,
-                             tr("Show/Hide Xsheet Camera Column"), "");
+  createMenuXsheetAction(MI_ToggleXsheetCameraColumn,
+                         tr("Show/Hide Xsheet Camera Column"), "");
```

Comment:
I think this can still be `createRightClickMenuAction` as the command is not included in the `Xsheet` menu.